src/main/scala/piconot/Empty.scala

```diff
@@ -0,0 +1,32 @@
+package piconot
+
+/**
+ * @author aputman
+ */
+object Empty extends PracticalBot {
+  picobot("empty") (
+    state ("toCorner") (
+      move(left),
+      move (up) (
+        check(left -> blocked)
+      ),
+      stay (
+        check(left -> blocked, up -> blocked) -> "toRight"
```

Comment:
Are these the sort of statements that will be eliminated when you add the "else" clauses?